tsMuxer/dvbSubStreamReader.cpp

```diff
@@ -38,8 +38,8 @@ int DVBSubStreamReader::intDecodeFrame(uint8_t* buff, uint8_t* end)
 {
     return 0;  // not working yet
 
-    unsigned pos, cmd, x1, y1, x2, y2, offset1, offset2, next_cmd_pos;
-    unsigned cmd_pos, is_8bit = 0;
+    int pos, cmd, x1, y1, x2, y2, offset1, offset2, next_cmd_pos;
+    int cmd_pos, is_8bit = 0;
```

Comment:
Note that this code is never executed (`return 0;  // not working yet`).
This code is a copy of [ffmpeg DVD subtitle decoder](https://github.com/FFmpeg/FFmpeg/blob/2f20e35892c866d4e141e73a3e2df16457f9a21a/libavcodec/dvdsubdec.c#L220), nothing to do with DV**B** subtitles. Maybe it is better to remove to avoid confusion and comply with GNU license.